packages/web-forms/src/components/controls/Input/InputControl.vue

```diff
@@ -35,13 +36,19 @@ provide('isInvalid', isInvalid);
 		<template v-else-if="node.valueType === 'string' && node.appearances.numbers">
 			<InputNumbersAppearance :node="node" />
 		</template>
+		<template v-else-if="node.valueType === 'geopoint'">
+			<InputGeopoint :question="node" />
+		</template>
 		<template v-else>
 			<InputText :node="node" />
 		</template>
 
-		<i v-show="isInvalid && (doneAnswering || submitPressed)" class="icon-error" />
+		<i v-show="isInvalid && (doneAnswering || submitPressed) && node.valueType !== 'geopoint'" class="icon-error" />
```

Comment:
Maybe add a comment for this exception? It took me a while to understand the intent of the change.

Longer term, I'm not really sure we want this structure: it seems like maybe each control (and maybe even specific appearances of some controls?) should determine whether to include this. I also wouldn't mind if a comment included a TODO note noting this!

Reply:
Done. I added a comment